src/core/stdcpp/array.d

```diff
@@ -0,0 +1,149 @@
+/**
+ * D header file for interaction with C++ std::array.
+ *
+ * Copyright: Copyright (c) 2018 D Language Foundation
+ * License: Distributed under the
+ *      $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost Software License 1.0).
+ *    (See accompanying file LICENSE)
+ * Authors:   Manu Evans
+ * Source:    $(DRUNTIMESRC core/stdcpp/array.d)
+ */
+
+module core.stdcpp.array;
+
+// hacks to support DMD on Win32
+version (CppRuntime_Microsoft)
+{
+    version = CppRuntime_Windows; // use the MS runtime ABI for win32
+}
+else version (CppRuntime_DigitalMars)
+{
+    version = CppRuntime_Windows; // use the MS runtime ABI for win32
+    pragma(msg, "std::array not supported by DMC");
```

Comment:
Shouldn't it be a `static assert(false, "...")` ?

Reply:
Then autotesters fail...

Reply:
That means it's running on Win32 even though it should not. Can you just disable the test there ?

Reply:
You got them running on Windows in the first place, didn't you ?

Reply:
I don't use OMF. I wish it would die already!

Reply:
DMD doesn't have `<array>`

Reply:
Okay, that one makes sense